runtime/genesis.go

```diff
@@ -4,43 +4,17 @@ import (
 	"encoding/json"
 	"os"
 
-	consTypes "github.com/pokt-network/pocket/consensus/types"
-	persTypes "github.com/pokt-network/pocket/persistence/types"
-	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/runtime/genesis"
 )
 
-var _ modules.GenesisState = &runtimeGenesis{}
-
-type runtimeGenesis struct {
-	ConsensusGenesisState   *consTypes.ConsensusGenesisState   `json:"consensus_genesis_state"`
-	PersistenceGenesisState *persTypes.PersistenceGenesisState `json:"persistence_genesis_state"`
-}
-
-func NewGenesis(
-	consensusGenesisState modules.ConsensusGenesisState,
-	persistenceGenesisState modules.PersistenceGenesisState,
-) *runtimeGenesis {
-	return &runtimeGenesis{
-		ConsensusGenesisState:   consensusGenesisState.(*consTypes.ConsensusGenesisState),
-		PersistenceGenesisState: persistenceGenesisState.(*persTypes.PersistenceGenesisState),
-	}
-}
-
-func (g *runtimeGenesis) GetPersistenceGenesisState() modules.PersistenceGenesisState {
-	return g.PersistenceGenesisState
-}
-func (g *runtimeGenesis) GetConsensusGenesisState() modules.ConsensusGenesisState {
-	return g.ConsensusGenesisState
-}
-
-func parseGenesisJSON(genesisPath string) (g *runtimeGenesis, err error) {
+func parseGenesisJSON(genesisPath string) (g *genesis.GenesisState, err error) {
```

Comment:
NIT: Can you rename this to `loadGenesis` and add a comment that `genesisPath` expects a JSON. 

`parseGenesisJSON(path)` feels incorrect.